b2

```diff
@@ -840,11 +843,29 @@ class B2RawApi(object):
         url = self._build_url(realm_url, 'b2_authorize_account')
         return post_json(url, {}, auth)
 
-    def create_bucket(self, api_url, account_id, account_auth_token, bucket_name, bucket_type):
+    def create_bucket(self, api_url, account_auth_token, account_id, bucket_name, bucket_type):
         url = self._build_url(api_url, 'b2_create_bucket')
-        params = {'accountId': account_id, 'bucketName': bucket_name, 'bucketType': bucket_type,}
+        params = {'accountId': account_id, 'bucketName': bucket_name, 'bucketType': bucket_type}
+        print url
+        print params
         return post_json(url, params, account_auth_token)
 
+    def delete_bucket(self, api_url, account_auth_token, account_id, bucket_id):
+        url = self._build_url(api_url, 'b2_delete_bucket')
+        params = {'accountId': account_id, 'bucketID': bucket_id}
+        return post_json(url, params, account_auth_token)
+
+    def list_buckets(self, api_url, account_auth_token, account_id):
```

Comment:
this could be refactored with `delete_bucket` and `update_bucket` which are almost identical